src/component/TipDisplay.spec.tsx

```diff
@@ -66,6 +66,33 @@ describe('TipDisplay component', () => {
     expect(hoverMessage.textContent).toContain('icons');
     expect(hoverMessage.textContent).toContain('quantities');
   });
+  test('should show only one tooltip at a time on mobile', () => {
+    const mockMobile = jest.fn();
+    mockMobile.mockReturnValue(true);
+
+    MobileCheck.isMobile = mockMobile;
+
+    const { getByAltText, getAllByText, queryAllByText } = render(<TipDisplay tip={11000 * 2} />);
+    const mediumBellBagIcon = getByAltText('Medium Bell Bag Icon');
+    const smallBellBagIcon = getByAltText('Small Bell Bag Icon');
+
+    // show  medium bell bags tooltip
+    fireEvent.click(mediumBellBagIcon);
+
+    const mediumBellBagTooltip = getAllByText(/stacks/i);
+    expect(mediumBellBagTooltip).toHaveLength(1);
+
+    // show small bell bags tooltip
+    fireEvent.click(smallBellBagIcon);
+
+    const smallBellBagTooltip = getAllByText(/stacks/i);
+    expect(smallBellBagTooltip).toHaveLength(1);
+
+    // hide small bell bags tooltip
+    fireEvent.click(smallBellBagIcon);
+
+    expect(queryAllByText(/stacks/i)).toHaveLength(0);
```

Comment:
I think this last assertion would provide more value if it were moved before the first bell bag is clicked. That way we can verify as a precondition check that there are no tooltips that appear when the page loads (which would be an error).

Reply:
I kept it both at the start and the end of the test, since no tooltips should be shown at those points